packages/SwingSet/src/lib/storageAPI.js

```diff
@@ -19,10 +19,9 @@ export function insistStorageAPI(kvStore) {
 }
 
 /**
- * Assert function to ensure that something expected to be an enhanced storage
- * object actually implements the enhanced storage API.  It should be a storage
- * object that additionally has the methods `enumeratePrefixedKeys`,
- * `getPrefixedValues`, and `deletePrefixedKeys`.
+ * Assert function to ensure that an object implements the enhanced storage API.
+ * (StorageAPI plus methods { enumeratePrefixedKeys, getPrefixedValues,
+ * deletePrefixedKeys }), also known as "KVStorePlus".
```

Comment:
Given that the enhancements all relate to number-suffixed keys, should "enhanced storage API" be renamed to something like "numericalKeyStorageAPI" or "StorageAPIPlusNumericalKeys"?

Reply:
Eh, I think that'd put too much detail into the name, it's not like we have a second enhancement layer that needs to be distinguished from this one.